packages/extension-align-div/src/align-div.ts

```diff
@@ -0,0 +1,41 @@
+import { Node, mergeAttributes } from '@tiptap/core';
+
+export const AlignDiv = Node.create({
+  name: 'aligndiv',
+  content: 'inline+',
+  marks: '',
+  group: 'block',
+
+  inline: false,
+  selectable: true,
+  draggable: true,
+  addOptions() {
+    return {
+      HTMLAttributes: {},
+    };
+  },
+  parseHTML() {
+    return [
+      {
+        tag: 'div[style*="text-align"]',
+      },
+    ];
+  },
+  addAttributes() {
+    return {
+      style: {
+        default: 'text-align: left',
+        parseHTML: (element) => {
+          return element.getAttribute('style');
+        },
+      },
+    };
+  },
+  renderHTML({ HTMLAttributes }) {
```

Comment:
Le render est ici fait sous forme de `<div>`, mais une autre stratégie serait possible :
=> faire un rendu compatible avec le format natif TipTap (en réalité, notre format "natif". C'est `<p>` pour le texte, voir commentaire ci-après)

Ainsi, l'extension convertirait les alignements "old-format" vers le nouveau format,
et elle serait utilisée par le transformateur uniquement, pas par les appli front.

Reply:
Autre possibilité : l'extension TipTap qui gère les alignements est apparemment configurable.
https://tiptap.dev/docs/editor/api/extensions/text-align#types

Il faudrait essayer de la configurer comme ça et tester pour voir : 
```suggestion
TextAlign.configure({
  renderHTML({ HTMLAttributes }) {
  types: ['div'],
})
```
L'inconvénient sera que le `renderHTML` donnera des `<div style="text-align:...">`, et qu'il faudra alors les gérer côté front (web et mobile)
Côté Web, on a déjà ça actuellement : 
`      TextAlign.configure({
        types: ["heading", "paragraph", "custom-image", "video"],
      }),
`